R/plot2.R

```diff
@@ -256,8 +266,10 @@ plot2.default = function(
     palette = NULL,
     legend = NULL,
     pch = NULL,
-    col = NULL,
     lty = NULL,
+    col = NULL,
```

Comment:
Should `col` accept "by"?

Reply:
It's a good question (and probably answered with "yes"). Initially, the default (and only) behaviour was for groups to be represented through colour. So `col="by"`was kind of redundant even after we added support for other graphical parameters. But ofc that doesn't mean it should error if users _do_ specify it manually.